mkdocs.yml

```diff
@@ -7,6 +7,13 @@ repo_url: https://github.com/MiBiPreT/anatrans/
 edit_uri: "edit/main/docs/"
 copyright: Copyright &copy; 2024 Alraune Zech, Sona Aseyednezhad, Robin Richardson, Jaro Camphuijsen
 
+nav:
```

Comment:
@AlrauneZ This is where you can define the navigation side bar for the docs site. You can give the heading you would like it to have in the side bar, and then the location of the markdown file that should be used for that page.